appscale/tools/templates/AppScalefile-cloud

```diff
@@ -1,13 +1,16 @@
 ---
 # The underlying cloud infrastructure to deploy AppScale over.
 # Set this to 'ec2' if running on Amazon EC2, 'euca' if running on Eucalyptus,
-# or 'gce' if running on Google Compute Engine.
+# 'gce' if running on Google Compute Engine, or 'azure' if running on
+# Microsoft Azure.
 infrastructure : 'ec2'
 
-# The Amazon or Eucalyptus Machine Image (ami or emi) that has AppScale
-# installed on it. When running on Google Compute Engine, set this to the name
-# of the image you set when you ran 'gcutil addimage'.
+# The Amazon / Eucalyptus Machine Image (ami or emi) or Azure image resource
+# url that has AppScale installed on it. When running on Google Compute Engine,
+# set this to the name of the image you set when you ran 'gcutil addimage'.
 machine : 'ami-XXXXXX'
+# machine: 'https://{storageaccount}.blob.core.windows.net/system/' +
```

Comment:
Yeah it is actually easy to get this URL directly from the Portal. I just wanted to break it down to provide a hint as to where the image captured could be in the Portal. I did think of passing the parameters in and constructing the url but the image could be linked from an existing storage account and the vm is now being created in another account. In which case, it would complicate our AppScalefile.